src/broker/Manager.cc

```diff
@@ -1,8 +1,9 @@
 #include "zeek/broker/Manager.h"
 
-#include <broker/broker.hh>
 #include <broker/config.hh>
 #include <broker/configuration.hh>
+#include <broker/endpoint.hh>
+#include <broker/variant.hh>
```

Comment:
Unrelated and not for this PR: I wonder if adding a `clang-format` rule to visually separate `<broker/...>` includes from `std` includes might be a nice cleanup.

Reply:
Agreed. Our default setup for zeek puts the standard headers first, followed by anything else. It should probably be the same for broker, but with broker headers instead of zeek headers.

Reply:
I'll do that as a followup.